src/classes/STG_SettingsManager_CTRL.cls

```diff
@@ -69,6 +69,18 @@ public with sharing class STG_SettingsManager_CTRL extends STG_Panel {
         installScript.runNewOrgScript();
         //Set master schedulables config
         UTIL_MasterSchedulableHelper.abortOldScheduledJobs();
-        UTIL_MasterSchedulableHelper.setScheduledJobs();
+        if (runningUserIsAdmin()) {
+            UTIL_MasterSchedulableHelper.setScheduledJobs();
+        }
+    }
+
+    /*******************************************************************************************************
+    * @description Returns whether the running user is a System Administrator.
+    * @return void
+    */
+    private boolean runningUserIsAdmin() {
+        Profile currentUserProfile = [SELECT Id, PermissionsCustomizeApplication, PermissionsModifyAllData 
```

Comment:
is there any chance the user running this query might not have perms on Profile and it would throw an exception?  maybe we should wrap the entire query in a try/catch, and return false if it throws?